testing/src/main/java/org/oppia/android/testing/RecyclerViewScrollingActions.kt

```diff
@@ -0,0 +1,50 @@
+package org.oppia.android.testing
+
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.scrollTo
+import androidx.test.espresso.contrib.RecyclerViewActions.actionOnItemAtPosition
+import androidx.test.espresso.contrib.RecyclerViewActions.scrollToPosition
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import javax.inject.Inject
+
+/**
+ * Provides the different scrolling options for RecyclerView
+ *
+ * This is needed because few Roboelectric tests require different scrolling actions for the tests
+ * to be successful.
+ * See https://github.com/oppia/oppia-android/issues/2209 for more information.
+ */
+class RecyclerViewScrollingActions @Inject constructor(
+  val testCoroutineDispatchers: TestCoroutineDispatchers
+) {
+
+  /**
+   * Returns a [ViewAction] which scrolls RecyclerView to a position
+   * Usage : TODO
+   */
+  fun scrollToPosition(viewId: Int, position: Int) {
+    onView(withId(viewId)).perform(
+      scrollToPosition<RecyclerView.ViewHolder>(
+        position
+      )
+    )
+    testCoroutineDispatchers.runCurrent()
+  }
+
+  /**
+   * Performs a [ViewAction] on a view at position.
+   * 1) Scroll RecyclerView to position
+   * 2) Perform an action on the view at position
+   * Usage : TODO
```

Comment:
Some thoughts:
- We probably don't care about documenting usage at this level (that can be introspected by seeing where this method is being used, instead)
- The documentation shouldn't discuss _how_ the method works, just what it's promising to do
- Between this & the method above, I don't understand when to pick one versus the other. How does one make that decision? These KDocs should clearly explain _when_ it's appropriate to use each one so that it can become clearer to others when trying to decide which to use when scrolling in a recycler view.

Reply:
Noted. KDoc for latest changes implemented with these changes.